lib/src/logic.dart

```diff
@@ -562,6 +562,9 @@ class Logic {
   /// Logical right-shift.
   Logic operator >>>(dynamic other) => RShift(this, other).out;
 
+  /// Unary clog2
```

Comment:
The doc comment here could be more helpful.  Somthing more like you had in the README.  Like "Ceiling of log base 2."  The fact that it's unary isn't so interesting like it is for `and()` where you may perform an and on one bus or two inputs.